sig-network/charter.md

```diff
@@ -0,0 +1,55 @@
+# SIG Network Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Network is responsible for the components, interfaces, and APIs which provide networking capabilities to Kubernetes
+users and workloads. SIG Network's goal is to provide simple, flexible abstractions on top of raw networking details, and to
+ensure Kubernetes provides the necessary networking features to enable application developers and cluster operators can
+meet their operational requirements.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Integration points with networking implementations (e.g. CNI integration)
+- Services / kube-proxy
+- Ingress
+- Network policy
+- DNS
+
+#### Cross-cutting and Externally Facing Processes
+
+### Out of scope
+
+- The CNI specification itself, which is maintained outside the Kubernetes project.
+- Particular implementations of the CNI specification.
```

Comment:
- Particular implementations of Ingress

It might also be worth adding "Particular implementations of NetworkPolicy" since it may not be immediately obvious to most readers that that falls under "particular implementations of the CNI spec"